src/pages/MovieDetail/MovieContent/CommentBox/AddComment/AddComment.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './AddComment.scss';
+
+class AddComment extends Component {
+
+
+  render() {
+    const { comments } = this.props;
+
+    return(
+      <>
+        <div className='commentAdd'>
+          {comments?.map((element) => {
+            return (
+              <div className='newComment'>{element.comment}</div>
+            )
```

Comment:
comments-comment로 변수 명 작성해주시고 key prop 작성해주세요
```suggestion
          {comments?.map(({ comment }, idx) => (
              <div className='newComment' key={idx}>{comment}</div>
            )
```
이렇게 하셔도 되겠네요